.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 21.11b1
```

Comment:
It would be nice to have a comment about how this version must match `requirements-tests-py3.txt`, ideally in both places. Same for isort.

Reply:
Good point, done